installInstructions/install3rdPartyPackages.rst

```diff
@@ -325,6 +325,14 @@ Note that the version of these libraries might be different on your machine :
    # If needed
    sudo apt-get install freeglut3-dev
 
+If the tk library is install but not found, you may need to add the include library to the CFLAGS used by make. 
+Likewise, if during compilation the linker cannot find definitions for sin and cos, you will also need to include the missing math library by passing ``-lm``.
+The ./configure script in CGNS looks at the environment variables to add user flags, so the final call to ./configure should look like: 
+
+.. prompt:: bash
+   
+   CFLAGS="-I/usr/include/tk -lm" LIBS="-lm" ./configure <....the rest>
+
 .. warning::
```

Comment:
Unrelated but could you remove this block since it's for 3.3.0 which is no longer supported?

Reply:
Aren't we still using 3.3.0 for stable?

Reply:
Oh shoot we haven't removed 3.3.0 yet, okay let's do that when we roll `stable` to 4.1.2.